redisinsight/api/scripts/default-guides.ts

```diff
@@ -0,0 +1,42 @@
+import * as path from 'path';
+import { join } from 'path';
+import * as config from '../src/utils/config';
+import { getFile, updateFolderFromArchive, updateFileFromArchive } from '../src/utils';
+
+const PATH_CONFIG = config.get('dir_path');
+const GUIDES_CONFIG = config.get('guides');
+
+const archiveUrl = new URL(path.join(
+  GUIDES_CONFIG.updateUrl,
+  GUIDES_CONFIG.zip,
+)).toString();
+
+const buildInfoUrl = new URL(path.join(
+  GUIDES_CONFIG.updateUrl,
+  GUIDES_CONFIG.buildInfo,
+)).toString();
+
+const buildInfoPath = join(PATH_CONFIG.defaultGuides, GUIDES_CONFIG.buildInfo);
+
+async function init() {
+  try {
+    // get archive
+    const data = getFile(archiveUrl);
```

Comment:
looks like you should `wait` for async function here and everywhere below in this file